salt/metalk8s/repo/online.sls

```diff
@@ -15,21 +15,28 @@ check that the system is registered:
     - name: subscription-manager status
 {%- endif %}
 
-Configure Kubernetes repository:
+Configure kubernetes repository:
   pkgrepo.managed:
-    - name: {{ repo.kubernetes.name }}
+    - name: kubernetes
     - humanname: Kubernetes
     - baseurl: "https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64"
     - gpgcheck: 1
     - repo_gpg_check: 1
     - gpgkey: "https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg"
-    - enabled: 0
+    - enabled: 1
```

Comment:
Same as above we should `enabled: 0`

Reply:
We can't do that, in online mode it's difficult to use a fromrepo since we don't really know which and how repositories are configured.